src/reducers/rootReducer.js

```diff
@@ -0,0 +1,38 @@
+const initialState = {
+additionalPrice: 0,
+car: {
+  price: 26395,
+  name: '2019 Ford Mustang',
+  image:
+    'https://cdn.motor1.com/images/mgl/0AN2V/s1/2019-ford-mustang-bullitt.jpg',
+  features: []
+},
+additionalFeatures: [
+  { id: 1, name: 'V-6 engine', price: 1500 },
+  { id: 2, name: 'Racing detail package', price: 1500 },
+  { id: 3, name: 'Premium sound system', price: 500 },
+  { id: 4, name: 'Rear spoiler', price: 250 }
+]
+};
+
+function rootReducer(state = initialState, action) {
```

Comment:
It does seem like you were understanding the logic! But this doesn't meet MVP fully we need to be taking an obj from additionalFeatures and placing it in car.features.